src/screens/SearchBeers/SearchBeers.jsx

```diff
@@ -83,6 +84,59 @@ const foodOptions = [
   },
 ]
 
+const yeastOptions = [
+  {
+    label: 'Pilsen Lager',
+    value: 'yeast=Pilsen Lager'
+  },
+  {
+    label: 'American Ale',
+    value: 'yeast=American Ale'
+  },
+  {
+    label: 'Belgian Ardennes',
+    value: 'yeast=Belgian Ardennes'
+  },
+  {
+    label: 'French Saison',
+    value: 'yeast=French Saison'
+  }
+]
+
+const hopsOptions = [
+  {
+    label: 'Fuggles',
+    value: 'hops=Fuggles'
+  },
+  {
+    label: 'First Gold',
+    value: 'hops=First Gold'
```

Comment:
Does api handle spaces? I don't think so  ("if you need to add spaces just add an underscore (_)" - it is from the documentation). Also you should not use capital letters - it should be like `first_gold`

Reply:
Done!